supportedBackends/google/pipelines/common/src/main/scala/cromwell/backend/google/pipelines/common/PapiPollResultMonitorActor.scala

```diff
@@ -0,0 +1,54 @@
+package cromwell.backend.google.pipelines.common
+
+import akka.actor.Props
+import cromwell.backend.google.pipelines.common.api.RunStatus
+import cromwell.backend.standard.pollmonitoring.{
+  AsyncJobHasFinished,
+  PollResultMessage,
+  PollResultMonitorActor,
+  ProcessThisPollResult
+}
+import cromwell.services.metadata.CallMetadataKeys
+
+import java.time.OffsetDateTime
+
+object PapiPollResultMonitorActor {
+  def props(tellMetadataFn: Map[String, Any] => Unit,
+            tellBardFn: (String, OffsetDateTime, Option[OffsetDateTime], OffsetDateTime) => Unit
+  ): Props = Props(new PapiPollResultMonitorActor(tellMetadataFn, tellBardFn))
+}
+
+class PapiPollResultMonitorActor(tellMetadataFn: Map[String, Any] => Unit,
+                                 tellBardFn: (String, OffsetDateTime, Option[OffsetDateTime], OffsetDateTime) => Unit
```

Comment:
Might be worth bringing back the `StartAndEndTimes` class, a series of `OffsetDateTime` can be easy to mess up.